<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: #D8D8D8; border-style: solid; border-width: 1px; border-color: #D8D8D8;</string>
  </property>
  <widget class="QWidget" name="settingWindow" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 49, 58);</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>502</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="5" column="1">
      <widget class="QPushButton" name="shortButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Short</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="sellButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Sell</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="longButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Long</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="buyButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Buy</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="predictButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>Predict</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
